Description: Fix AC_INIT handling
Author: Jörg Frings-Fürst <debian@jff.email>
Bug: <bug-libunistring@gnu.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=992281
Forwarded: yes 
Last-Update: 2021-08-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/configure.ac
===================================================================
--- configure.ac.orig
+++ configure.ac
@@ -16,12 +16,14 @@ dnl along with this program.  If not, se
 
 dnl Process this file with autoconf to produce a configure script.
 
+
+
 AC_PREREQ([2.63])
-AC_INIT
+. $srcdir/version.sh
+AC_INIT([libunistring],[m4_esyscmd_s([$VERSION_NUMBER])])
 AC_CONFIG_SRCDIR([lib/unistring/version.in.h])
 AC_CONFIG_AUX_DIR([build-aux])
-. $srcdir/version.sh
-gl_INIT_PACKAGE([libunistring], [$VERSION_NUMBER])
+dnl AM_INIT_AUTOMAKE
 AM_INIT_AUTOMAKE([silent-rules dist-xz])
 AM_CONFIG_HEADER([config.h])
 
